app/views/products/_review.html.erb

```diff
@@ -0,0 +1,7 @@
+<li class="list-group-item">
+  <%= hidden_field_tag :stars, review.score,
+    class: "rating", data: {filled: "fa fa-star fa-1x", empty: "fa fa-star-o fa-1x", readonly: true} %>
+  <strong> <%= review.user.name %></strong>
+  <p><small><%= l review.updated_at, format: :long %></small></p>
```

Comment:
1 là cái gì thế em???

Reply:
@tuanhv-0694 chỗ này là để in ra định dạng giờ a ạ. Nếu bỏ đi thì nó ra sẽ hiển thị giờ UTC nên em vẫn để lại ạ.